engine/access/rest/websockets/data_providers/base_provider.go

```diff
@@ -48,12 +71,73 @@ func (b *baseDataProvider) Topic() string {
 
 // Arguments returns the arguments associated with the data provider.
 func (b *baseDataProvider) Arguments() models.Arguments {
-	return b.arguments
+	return b.rawArguments
 }
 
 // Close terminates the data provider.
 //
 // No errors are expected during normal operations.
```

Comment:
not your changes, but no need to define errors when the function doesn't return any
```suggestion
```